protos/core.proto

```diff
@@ -26,22 +26,62 @@ message ProjectConfig {
   string gcloud_project_id = 6;
 }
 
-message Environment {
-  string name = 1;
+message Environments {
+  message Environment {
+    string name = 1;
+    
+    message GitReference {
+      oneof git_reference {
+        string branch = 1;
+        string commit_sha = 2;
+      }
+    }
+  
+    GitReference git_reference = 2;
+    ProjectConfig config_override = 3;
+
+    message Schedule {
+      // Required fields.
+      string name = 1;
+      string cron = 2;
+      repeated string tags = 3;
   
-  message GitReference {
-    oneof git_reference {
-      string branch = 1;
-      string commit_sha = 2;
+      // Optional.
+      bool include_dependencies = 4;
+      bool full_refresh = 5;
+      bool disabled = 6;
+  
+      message NotificationRequirement {
```

Comment:
ditto, writing `dataform.Environments.Environment.Schedule.NotificationRequirement` is going to be annoyingly verbose.
can we make Schedule a top level thing?
I think a new namespace would help a lot here